### ISSUE TITLE
Expose version via command line

### ISSUE THREAD

**Issue by mzupan** [projectcalico/calico-cni#56]
We bootstrap kubernetes and calico via salt. We do things like this for the `kubectl` app

```
curl -L https://storage.googleapis.com/kubernetes-release/release/v1.1.7/bin/linux/amd64/kubectl -o /usr/bin/kubectl:
  cmd.run:
    - unless: kubectl version | grep Client | grep v1.1.7
    - watch_in:
      - service: kubelet
```

It would be create if the `calico` and `calico-ipam` exposed a way to get the version that is currently installed on the system via like an argument 

```
calico --version
```

or 

```
calico version
```

Thanks!

**Comment by caseydavenport** [comment]
@mzupan I think a command line argument like this makes sense.

I know we've used docopt in other Calico projects to parse command line options, but we don't have anything in calico-cni yet since there are no command line options as of yet :)

As part of this we need to make sure the correct tooling / checks / automation are in place such that we can't accidentally make a calico-cni release without updating the reported version.

**Comment by caseydavenport** [comment]
@mzupan This is available in release v1.1

```
calico -v
calico --version
```
